cxx/distributions/crp.hh

```diff
@@ -25,6 +25,8 @@ class CRP {
 
   int sample(std::mt19937* prng);
 
+  double logp_new_table() const;
```

Comment:
Add description please.

Reply:
Nothing else in this class has descriptions and this is self-explanatory so I'll leave it.